src/managers/GeneralManager.ts

```diff
@@ -1,100 +1,51 @@
 import { MasterTick } from "core/masterTick";
 import { Nullable, UUID } from "definitions/utils";
 import { EventEmitter } from "utilities/EventEmitter";
-import {
-    DataInformation,
-    PlayerInformation,
-    SessionInformation,
-    TileInformation,
-} from "definitions/information";
-import {
-    playersNotEqual,
-    sessionsNotEqual,
-    tilesNotEqual,
-} from "utilities/information";
 import { api } from "../utilities/api";
 import axios from "axios";
 import PlayerManager from "./PlayerManager";
-import { assert } from "utilities/utils";
+import { DataDTO, SessionDTO, PlayerDTO, TileDTO } from "definitions/dto";
 
 class GeneralManager {
     public readonly onSync: EventEmitter;
-    private data: Nullable<DataInformation>;
+    private data: Nullable<DataDTO>;
 
     public constructor() {
         this.onSync = new EventEmitter();
         this.data = null;
-        this.fetchData();
     }
 
-    private fetchData(): void {
-        MasterTick.on(async () => {
-            const requestBody: Nullable<UUID> = PlayerManager.getId();
-            if (requestBody === null) {
-                return;
-            }
-            const response: axios.AxiosResponse<DataInformation> =
-                await api.post("/ping", requestBody);
-            if (!this.data) {
-                this.data = response.data;
-                this.onSync.emit();
-                return;
-            }
-            if (this.checkUpdate(response.data)) {
-                this.data = response.data;
-                this.onSync.emit();
-            }
-        });
+    public async initialize(): Promise<void> {
+        await this.fetchData();
```

Comment:
why not just call this.ping()?

Reply:
Because fetch data does more then just call ping.